src/components/ButtonGroup.js

```diff
@@ -0,0 +1,101 @@
+const React = require('react');
+const Radium = require('radium');
+
+const Button = require('./Button');
+
+const StyleConstants = require('../constants/Style');
+
+const ButtonGroup = React.createClass({
+  propTypes: {
+    buttons: React.PropTypes.arrayOf(React.PropTypes.shape({
+      icon: React.PropTypes.string,
+      text: React.PropTypes.string
+    }).isRequired),
+    controlArrows: React.PropTypes.bool,
+    primaryColor: React.PropTypes.string,
+    type: React.PropTypes.oneOf([
+      'base',
+      'disabled',
+      'neutral',
+      'primary',
+      'primaryOutline'
+    ])
+  },
+
+  getDefaultProps () {
+    return {
+      buttons: [],
+      controlArrows: false,
+      primaryColor: StyleConstants.Colors.PRIMARY,
+      type: 'primaryOutline'
+    };
+  },
+
+  _getArrows () {
+    this.props.buttons.unshift({ icon: 'caret-left' });
+    this.props.buttons.push({ icon: 'caret-right' });
+  },
+
+  render () {
+    const styles = this.styles();
+    const buttonType = this.props.type;
+    const primaryColor = this.props.primaryColor;
+
+    return (
+      <div {...this.props}>
+        {this.props.controlArrows ? this._getArrows() : null}
+        {this.props.buttons.map(function (button, index, arr) {
```

Comment:
Since I get undefined on props inside this method (I'm assuming this is because `this` is being scoped to the method?) I created a const and went about it that way.